comid/test_vars.go

```diff
@@ -516,6 +516,7 @@ var (
 			]
 		}
 	]
+ }
 }
-}`
+`
```

Comment:
this change looks spurious

Reply:
This is how it is in the latest commit, so I just retained it!